### PR TITLE
Porting Python PostCommit to Gradle.

### PR DESCRIPTION
**Pull request by pabloem**
Breaking down the postcommit into multiple gradle tasks, and reducing the size of the shell script
r: @udim

### CODE REVIEW

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by pabloem** [comment]
The localWordCount test is having trouble with importing fn api proto files. Strange.

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by pabloem** [comment]
Per https://stackoverflow.com/questions/19569557/pip-not-picking-up-a-custom-install-cmdclass, it seems that we were not translating all of the command classes properly to generate protobuf files.

**Comment by pabloem** [comment]
Local wordcount ran successfully : )

**Comment by pabloem** [comment]
Tests succeeded but failure related to posting status:
image

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by pabloem** [comment]
Run Python PostCommit

File: build.gradle [hunk 1]
```diff
@@ -159,16 +159,12 @@ task goPostCommit() {
...
 }
 
 task pythonPostCommit() {
-  doLast {
-    exec {
-      executable 'sh'
-      args '-c', './sdks/python/run_postcommit.sh'
-    }
-  }
+  dependsOn ":beam-sdks-python:preCommit"
```
Comment (udim) [inline build.gradle]: Redundant, since postCommit already dependsOn preCommit (in sdks/python/build.gradle).

Comment (pabloem) [reply build.gradle]: Good catch. Thanks!

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by pabloem** [comment]
PostCommit that's running:
https://builds.apache.org/job/beam_PostCommit_Python_Verify/4912/console

**Comment by pabloem** [comment]
PostCommit passed: https://builds.apache.org/job/beam_PostCommit_Python_Verify/4912/console

**Comment by pabloem** [comment]
@udim if this looks okay, I will merge it

**Comment by udim** [comment]
LGTM

**Pull request merged**